### PR TITLE
[DOC]Add a description field to examples

### LINKED ISSUE #4369: [DOC] Add a description field to examples’ json files
Document #4368

### PR DESCRIPTION
**Pull request by svekars**
Closes #4369

### CODE REVIEW

File: examples/ml/hyperparameter/model.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "pipeline": {
     "name": "model"
   },
+  "description": "A pipeline that trains the model for each combinataion of C and Gamma parameters.",
```
Comment (JoeyZwicker) [inline examples/ml/hyperparameter/model.json]: typo

File: examples/ml/hyperparameter/test.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "pipeline": {
     "name": "test"
   },
+  "description": "A pipeline that generates evaluation metrics by using each of the train models.",
```
Comment (JoeyZwicker) [inline examples/ml/hyperparameter/test.json]:
```suggestion
  "description": "A pipeline that scores each of the trained models.",
```

File: examples/ml/tensorflow/training_and_export.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "pipeline": {
     "name": "checkpoint"
   },
+  "description": "A pipeline that performs image-to-image translation in the `training` mode to output the results in the `checkpoint` repository and then runs the same code in the `export` mode against the results in the `checkpoint` repo to output the model to the `model` repository.",
```
Comment (JoeyZwicker) [inline examples/ml/tensorflow/training_and_export.json]: too long. Can we shorten this?

File: examples/ml/tensorflow/training_and_export_gpu.json [hunk 4]
```diff
@@ -2,6 +2,7 @@
   "pipeline": {
     "name": "checkpoint"
   },
+  "description": "A pipeline that performs image-to-image translation in the `training` mode to output the results in the `checkpoint`  repository and then runs the same code in the `export` mode against the results in the `checkpoint` repo to output the trained model to the `model` repository. This pipeline includes GPU configuration settings.",
```
Comment (JoeyZwicker) [inline examples/ml/tensorflow/training_and_export_gpu.json]: too long. Can we shorten this?

File: examples/ml/hyperparameter/model.json (hunk 1 above)
Comment (svekars) [reply examples/ml/hyperparameter/model.json]: addressed

File: examples/ml/hyperparameter/test.json (hunk 2 above)
Comment (svekars) [reply examples/ml/hyperparameter/test.json]: added

File: examples/ml/tensorflow/training_and_export.json (hunk 3 above)
Comment (svekars) [reply examples/ml/tensorflow/training_and_export.json]: shortened

File: examples/ml/tensorflow/training_and_export_gpu.json (hunk 4 above)
Comment (svekars) [reply examples/ml/tensorflow/training_and_export_gpu.json]: shortened

**Pull request merged**